### PR TITLE
Variant QC allele counts 

### PR DESCRIPTION
**Pull request by jkgoodrich**
Add computation of allele counts to be done by the default_compute_info` in gnomad_methods instead of as an additional pipeline step.

### CODE REVIEW

**Review by mike-w-wilson — APPROVED** [review]
One comment on an arg name I found a bit confusing but otherwise looks good to me!

File: gnomad_qc/v4/resources/basics.py [hunk 1]
```diff
@@ -164,15 +173,28 @@ def _remove_ukb_dup_by_index(
                 filter_ht = hard_filtered_samples.versions[CURRENT_VERSION].ht()
             else:
                 filter_ht = hard_filtered_samples_no_sex.versions[CURRENT_VERSION].ht()
-            vds = hl.vds.filter_samples(vds, filter_ht, keep=keep_samples)
 
-    if release_only:
+            filter_s = filter_ht.s.collect()
+            if high_quality_only_keep_controls:
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/resources/basics.py]: I decided to just change it to `keep_controls` and make it fully function that way. Let me know if it looks OK

File: gnomad_qc/v4/resources/basics.py [hunk 2]
```diff
@@ -189,7 +189,10 @@ def _remove_ukb_dup_by_index(
         else:
             meta_ht = meta.versions[CURRENT_VERSION].ht()
         if release_only:
-            vds = hl.vds.filter_samples(vds, meta_ht.filter(meta_ht.release))
+            filter_expr = meta_ht.release
+            if keep_controls:
+                filter_expr |= hl.literal(TRUTH_SAMPLES_S).contains(meta_ht.s)
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/resources/basics.py:194]: Neat -- I did not know this existed!

**Review by mike-w-wilson — APPROVED** [review]
LGTM

**Pull request merged**